src/commands.ts

```diff
@@ -148,22 +175,51 @@ const unsubCmd = new Command(
     if (!alias) {
       bot.sendMessage(
         channel,
-        `You need to provide the name of the game you want to unsubscribe from.\n`
-        + `Try ${unsubCmd.getTriggerLabel(channel)}.`,
+        `You need to provide the name of the game you want to unsubscribe from.\n` +
+          `Try ${unsubCmd.getTriggerLabel(channel)}.`,
       );
     }
 
-    for (const game of Game.getGames()) {
-      if (game.hasAlias(alias)) {
-        if (bot.removeSubscriber(channel, game)) {
-          bot.sendMessage(channel, `You are now unsubscribed from the **${game.label}** feed!`);
-        } else {
-          bot.sendMessage(
-            channel,
-            `You have never subscribed to the **${game.label}** feed in the first place!`,
-          );
+    const aliases = alias.split(', ');
+
+    // The games matching the alias
+    let aliasGames: Game[] = [];
+
+    for (alias of aliases) {
+      aliasGames = aliasGames.concat(Game.getGamesByAlias(alias));
+    }
+
+    if (aliasGames.length > 0) {
+      // The map of which game is a new sub
+      const gameMap = aliasGames.map((game) => {
+        const isNew = bot.removeSubscriber(channel, game);
+        return { game, isNew };
+      });
+
+      const newUnsubs = gameMap.filter((map) => map.isNew).map((map) => map.game);
+      const oldUnsubs = gameMap.filter((map) => !map.isNew).map((map) => map.game);
```

Comment:
`newUnsubs` and `oldUnsubs` sound a bit misleading, it took me a second to realize what they represent. Maybe `validUnsubs` and `invalidUnsubs`?

Reply:
True, fixed.